image/share/Taskfile

```diff
@@ -0,0 +1,3 @@
+default: KMER=121 PARTITIONS=4
+fast: KMER=63 PARTITIONS=128
+faster: KMER=31 PARTITIONS=256
```

Comment:
I've experimented with different parameter settings but I'm still getting similar contigs produced. The set of reads produces contigs when run through velvet. Perhaps @jjcook might be able to identify what I'm doing wrong?